doc/nrf/releases/release-notes-latest.rst

```diff
@@ -24,12 +24,25 @@ The Zephyr fork in |NCS| contains all commits from the upstream Zephyr repositor
 
 The following list summarizes the most important changes inherited from upstream Zephyr:
 
+* Architectures:
+
+  * Fixed parsing of Cortex-M MemManage Stacking Errors, to correctly report thread stack corruptions.
+
 * Kernel:
 
   * The ``CONFIG_KERNEL_DEBUG`` Kconfig option, which was used to enable ``printk()`` based debugging of the kernel internals, has been removed.
     The kernel now uses the standard Zephyr logging API at DBG log level for this purpose.
     The logging module used for the kernel is named ``os``.
 
+* Boards:
+
+  * Introduced board definition for nRF5340 DK.
+  * Added the 'netif' capability for nRF52840 DK.
+  * Added GPIO as a supported feature in nRF-based board definitions.
+  * Replaced I2C1 and SP2 pin assignments with D11-D13 in nRF5340 PDK.
```

Comment:
Did you mean SPI2?

Is the D11-D13 notation borrowed from Arduino? Should this be done in Px.y style instead?

Reply:
Yeap, SPI2, thanks.

Reply:
both notations are present on the PDK.
D11 - D13 correspond to P1.13 - P1.15. I can change that, as well.

Reply:
Actually, @mbolivar-nordic , I prefer to not list the pin numbers, but rather explain what the change was about.

new text:
```
Modified I2C1 and SPI2 pin assignments in nRF5340 PDK to match the
    standard location for I2C and SPI in the Arduino header.
```

Reply:
>  I prefer to not list the pin numbers, but rather explain what the change was about.

I think doing both would be better (as a user, I would rather just see the new assignments instead of having to google the arduino locations), but that's a minor point.